openquake/calculators/tests/classical_test.py

```diff
@@ -658,9 +658,9 @@ def test_case_78(self):
             hazard_calculation_id=hc_str)
 
         # skipped because broken on CI due to a mysterious extra space
```

Comment:
Guess this change is unrelated to the current PR, but if this is no longer skipped we should remove the comment above that says it is skipped

Reply:
Yep, done